tests/cli/test_cli.py

```diff
@@ -393,6 +394,9 @@ def __enter__(self):
         self.worksheets.append(temp_worksheet)
         _run_command([cl, 'work', temp_worksheet])
         self.disk_quota = _run_command([cl, 'uinfo', '-f', 'disk']).split(' ')[2]
+        self.time_quota = (
+            _run_command([cl, 'uinfo', '-f', 'time']).split(' ')[2].split('y')[0] + 'y'
```

Comment:
This is because of how the [parse duration](https://github.com/codalab/codalab-worksheets/blob/master/codalab/lib/formatting.py#L130) is written. Basically, it expects a string of the form "<number>[<s|m|h|d|y>]" and the default time quota for the root user is '100y0d'. Thus, I'm just removing the '0d' from the string. In general, this will just keep the number of years in the time quota.